repository/src/main/java/org/apache/atlas/tasks/TaskRegistry.java

```diff
@@ -225,6 +224,10 @@ public void complete(AtlasVertex taskVertex, AtlasTask task) {
         }
 
         updateStatus(taskVertex, task);
+
+        graph.commit();
```

Comment:
why is this needed?

Reply:
This was just making sure graph is committed at the end but it is invoked in later operations as well. I will delete this one.